### ISSUE TITLE
Incorrect Metadata/ Projection?

### ISSUE THREAD

**Issue by kurtjoy** [Element84/earth-search#39]
Found a few granules with weird metadata.
if I do a request like:

`https://earth-search.aws.element84.com/v1/collections/sentinel-2-c1-l2a/items?limit=12&datetime=2019-12-31T11:00:00.000Z/2020-01-14T11:00:00.000Z&bbox=173.4850878306321,-41.32156050393732,173.53181118851415,-41.30147765654245`

It returns a number of tiles in the T01 UTM zone that are not in the UTM 60/59/58 Id expect for New Zealand. When I looked at the metadata for one of these anomalous granules.

https://earth-search.aws.element84.com/v1/collections/sentinel-2-c1-l2a/items/S2A_T01GBQ_20200123T220614_L2A

It shows that the projection is 32701 with a centroid of -65, -0.0058, and basically defines an area from NZ down to -90 lat and 360 of long

![image](https://github.com/Element84/earth-search/assets/40187977/19143b7d-e7f4-486e-a395-2dc4424d9781)

Cheers

Kurt

**Comment by kurtjoy** [comment]
Should mention that the granule I gave as an example should be located here. (T01GBQ).
![image](https://github.com/Element84/earth-search/assets/40187977/783a101a-3ad4-4385-99d6-cc6ae042a9de)

**Comment by JonDHo** [comment]
This appears to be the always-painful antimeridian issue. It seems that items which cross over antimeridian are split into multiple polygons and some of them wrap the wrong way around the globe. The effect of this is that these items show up in searches which are unrelated. For example, I am searching for data in Chile, but get this item: https://earth-search.aws.element84.com/v1/collections/sentinel-2-c1-l2a/items/S2A_T01LAJ_20211023T222802_L2A

The geometry of this item looks like this:
![image](https://github.com/Element84/earth-search/assets/3302774/371eb72f-2154-4d60-9170-67a7e3d6cedc)

and zoomed in:
![image](https://github.com/Element84/earth-search/assets/3302774/9bd18b0f-0af2-4baa-8eb9-2b05fbda8b05)

It is clear that the item is meant to straddle the antimeridian, but the lower part of the geometry wraps around the globe.

Other examples include:
- https://earth-search.aws.element84.com/v1/collections/sentinel-2-c1-l2a/items/S2B_T01LAJ_20211117T222754_L2A
- https://earth-search.aws.element84.com/v1/collections/sentinel-2-c1-l2a/items/S2B_T01LAJ_20211028T222758_L2A
- https://earth-search.aws.element84.com/v1/collections/sentinel-2-c1-l2a/items/S2A_T01LAJ_20211112T222758_L2A

**Comment by kurtjoy** [comment]
Yeah. I'm having to do some wrangling post pystac and pre stackstac to
check if an AOI is one of the multipolygons or the other. Seems to work
fine, but still need to do some more logic if an AOI is across the
Antimeridian (Fiji I'm looking at you!)

**Comment by JonDHo** [comment]
@kurtjoy if you use shapely, then the is_valid property will return False for these entries, so that can help to identify the problematic ones.

**Comment by ircwaves** [comment]
stactools-packages/sentinel2#192 updates antimeridian handling, @tylanderson -- have you tested the problematic scenes identified here with that update?

**Comment by tylanderson** [comment]
> [stactools-packages/sentinel2#192](https://github.com/stactools-packages/sentinel2/pull/192) updates antimeridian handling, [@tylanderson](https://github.com/tylanderson) -- have you tested the problematic scenes identified here with that update?

I've tested these and a few other cases locally and found them to be resolvable. We will still need to republish the 500+ scenes that have these issues in the catalog

**Comment by tylanderson** [comment]
This has been resolved with the latest updates to earth search, with any previously failing antimeridian crossing scenes reprocessed and available

Example:
https://earth-search.aws.element84.com/v1/collections/sentinel-2-c1-l2a/items/S2A_T01GBQ_20200123T220614_L2A
Viewer Link:
https://geojson.io/#data=data:text/x-url,https%3A//earth-search.aws.element84.com/v1/collections/sentinel-2-c1-l2a/items/S2A_T01GBQ_20200123T220614_L2A

![Image](https://github.com/user-attachments/assets/fee2d57f-215e-40f0-b641-697a43b7ffb5)
